pkg/csi/cinder/openstack/openstack.go

```diff
@@ -61,13 +63,27 @@ type Config struct {
 		Password   string
 		TenantId   string `gcfg:"tenant-id"`
 		TenantName string `gcfg:"tenant-name"`
+		TrustID    string `gcfg:"trust-id"`
 		DomainId   string `gcfg:"domain-id"`
 		DomainName string `gcfg:"domain-name"`
 		Region     string
 		CAFile     string `gcfg:"ca-file"`
 	}
 }
 
+func logcfg(cfg Config) {
+	klog.V(5).Infof("AuthURL: %s", cfg.Global.AuthUrl)
```

Comment:
Why bothering adding these creds into logs?

Reply:
because it's easier to debug then. this code part is present in the controller image also.
If I would have had them, I would'nt had to use ngrep between the kubernetes and my openstack controller to figure out that the trust-id wasn't transmitted I would have seen it in the logs.

this also helps to see that your cloud-config file is correct. There is no password, so probably no security issues.

Reply:
we could move to common place as used in multiple places . may be pkg/util/util.go?

Reply:
outputting the configuration/flags is pretty standard behavior for V(4) in Kubernetes.. fwiw
we talked about this in the OS Summit in Denver. The loadbalancers, csi, keystone, and kms we've all made basically use the same bit of code downstream already, might as well make it a importable utility/library instead.